Makefile

```diff
@@ -308,6 +308,30 @@ else
 	$(QUIET)mv $(PDFDIR)/$(CSPEC)-optimized.pdf $@
 endif
 
+# C++ for OpenCL doc
+EXTSPEC = OpenCL_CXX4OpenCL
+EXTSPECSRC = $(EXTSPEC).txt $(GENDEPENDS) \
```

Comment:
We need to name this document something else other than "EXTSPEC", with different targets than "exthtml" and "extpdf" since they're already used for the extensions document.

I'm not a huge fan of "OpenCL_CXX4OpenCL" either - could this just be "CXX_for_OpenCL" instead?  I know this is a "bike shedding" comment, so not going to lose a lot of sleep either way.